server/controllers/PoliticalParty.js

```diff
@@ -67,6 +67,37 @@ class PartiesController {
       data: [findParty],
     });
   }
+
+  static editPartyName(req, res) {
+    const { partyid, name } = req.params;
+    if (name === 'name') {
```

Comment:
Why are you checking if `(name === 'name')`

Reply:
its is to check if the last part of the parameter in the url is name.
since a sample request should be PATCH  api/v1/parties/2/name